apps/trading/e2e/tests/wallet/test_wallet.py

```diff
@@ -91,6 +102,9 @@ def test_wallet_connection_rejected(continuous_market, page: Page):
     expect(page.get_by_test_id("dialog-content").nth(1)).to_have_text(
         "User errorthe user rejected the wallet connectionTry againAbout the Vega wallet  | Supported browsers "
     )
+    expect(page.get_by_test_id("dialog-content").nth(1)).to_have_text(
```

Comment:
This is a duplication of the line above

Reply:
Yeah so this is the crappy "fix"

Playwright doesn't acknowledge errors thrown within routing, so the test will "error" but the test result will be pass, 

But duplicating the expect the test is then failing correctly.

There must be a better solution but this one works.

I'm more than happy for someone else to have a look to see if they can solve better.